e2e/fixtures/auth.setup.ts

```
@@ -13,40 +13,116 @@ if (!fs.existsSync(authDir)) {
+    await page.waitForFunction(
+      () => !window.location.pathname.includes('/auth/authorize'),
+      { timeout: 60000 }
+    );
+
+    // Wait for home-assistant element to appear (indicates successful login)
+    await page.waitForSelector('home-assistant', { timeout: 30000 });
+
+    // Wait for HA to fully initialize
+    await page.waitForFunction(
+      () => {
+        const ha = document.querySelector('home-assistant') as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Expected '!==' and instead saw '!='.

(eqeqeq)